index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div id="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <div class="menu-items">
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div id="burger">
+        <img src="/images/icon-Menu.svg" alt="burger_icon" />
+      </div>
+    </nav>
```

Comment:
- [ ] Wonderful job🎉🎉 incorporating the nav tag! 
- [ ] However, to adhere to [HTML semantic conventions](https://www.freecodecamp.org/news/semantic-html5-elements/) more closely, it would be fantastic if you could include a `header` tag and place the `nav` tag inside it. 
- [ ] Additionally, it seems that the `hamburger` image is not displaying in the user interface. Could you kindly ensure that the menu image is visible?
- [ ]  I appreciate you including the logo within the `anchor` tag – it greatly enhances the visual appeal of your design and aligns with the project requirements
- [ ] your design 
![your](https://github.com/Emeryherve/setup-and-mobile-first/assets/118761303/54a1cd7b-b7b5-45ef-890c-d1e63ace7d4a)
Figma:
![mobile](https://github.com/Emeryherve/setup-and-mobile-first/assets/118761303/1c050647-9f5c-4f79-be5a-bb82120723d4)